src/cache/chrome-storage-assignment-cache.ts

```diff
@@ -0,0 +1,31 @@
+import { AssignmentCacheKey } from '@eppo/js-client-sdk-common/dist/cache/assignment-cache';
+
+import ChromeStorageAsyncMap from './chrome-storage-async-map';
+import { ReadableAssignmentCache } from './hybrid-assignment-cache';
+
+export default class ChromeStorageAssignmentCache implements ReadableAssignmentCache {
```

Comment:
@leoromanovsky should we just export this class and let users use it with `useCustomAssignmentCache` if they want, or we want to use this in one of the existing code paths?

Reply:
Sorry I was referring to `HybridAssignmentCache` specifically, but same question